### ISSUE TITLE
New crates.io release

### ISSUE THREAD

**Issue by dabreegster** [urschrei/lonlat_bng#20]
https://docs.rs/lonlat_bng/latest/lonlat_bng/ and crates.io point to 0.3.8, which is ancient! I actually thought this crate was broken (valid inputs to `convert_osgb36_to_ll` all failed), but it's working wonderfully when pinning to git. For anyone just doing `cargo add lonlat_bng`, a new release would be helpful! Thanks for this crate and saving me a GDAL dependency, by the way :)

**Comment by dabreegster** [comment]
Ooh, that explains it. I don't have ideas that would keep the crate hermetic. (Hosting the data elsewhere and lazily pulling it in would suddenly introduce a dependency on the network at runtime, and require stable hosting somewhere.)

Worth keeping the issue open for the next person who gets confused and winds up here?

**Comment by urschrei** [comment]
An alternative is using proj and enabling its network feature, which enables the use of downloaded grid shift files: the OSTN15 NTv2 grid shift is definitely available. That may not be an option for you if you need to build your crate as WASM. In that case you should talk to Thomas about using the [OSTN15 NTv2 grid shift file](https://www.ordnancesurvey.co.uk/business-government/tools-support/os-net/for-developers) in the geodesy crate: it has NTv2 support and can be compiled to wasm.

But yes, I'll keep this open.

**Comment by urschrei** [comment]
> With the [release of v0.8.0 to crates.io](https://crates.io/crates/lonlat_bng/0.8.0) this can be closed. 🥳

Yes, and many thanks for a much cleaner solution to the grid source issue than what we had before. In my defence it was 2015 and nobody knew how to do anything in Rust.
